fceciliano/TrainingFabian
# Feature/task 5

# Pull Request
## Description
This task consists to do the deploy of the previous task in which the home page of One Piece figma proyect was developed. This deploy was made using git pages using this [guide](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) and some tutorials.

Feature (Task #5)

## Type of change

## How Has This Been Tested?
To verify this task:
- You must go to [this](https://fceciliano.github.io/TrainingFabian/?p=all) link, which contain the exactly project development previously, here you can see all the components in the seccions of “Atoms”, “Molecules”, “Organisms”, “Templates” and “Pages”
